docs/developer/local-docker.md

```diff
@@ -50,6 +59,97 @@ This will launch the integration test  runner against the CI environment, using
 
 This may all be overridden.
 
+## Unit tests
+
+Unit tests require a running Narrative server in order to access Jupyter javascript files which are only available when the server is built and running.
+
+As a server, it can be accessed within a container is well, and for those who prefer Docker-based workflows, is quite convenient.
```

Comment:
> within a container _as_ well

Reply:
(I think that's what it's supposed to be, anyway!)

Reply:
If you can believe it, I'm back on this PR, nearly a year to the day 🙀 
And, yes, is -> as.